src/proposals_dlg.py

```diff
@@ -1606,7 +1606,7 @@ def read_voting_from_network(self, force_reload_all, proposals: List[Proposal]):
                         db_oper_duration = 0.0
                         db_oper_count = 0
                         network_duration = 0.0
-                        node_info = self.dashd_intf.rpc_call(False, False, 'getinfo')
+                        node_info = self.dashd_intf.rpc_call(False, False, 'getnetworkinfo')
                         if node_info.get('version', 140000) < 140000:
```

Comment:
Since the comparison, to the v 14000 is no longer needed (it's an ancient history), this call will be removed in the next release.